components/blocks/video.tsx

```diff
@@ -33,6 +29,7 @@ export const videoBlockSchema: Template = {
     },
   },
   fields: [
+    sectionBlockSchemaField as any,
```

Comment:
```suggestion
    sectionBlockSchemaField as any,
```
nitpick: I think there should be a type we can make `sectionBlockSchemaField` so we don't need to cast this as `any`